dataverse-webapp/src/test/java/edu/harvard/iq/dataverse/export/openaire/FirstNamesTest.java

```diff
@@ -22,38 +27,25 @@ public void testFirstName() {
         assertTrue(firstNames.isFirstName("Lainey"));
     }
 
-    /**
-     * Name is composed of:
-     * <First Names> <Family Name>
-     */
-    @Test
-    public void testName() {
-        assertEquals(firstNames.getFirstName("Jorge Mario Bergoglio"), "Jorge Mario");
-        assertNull(firstNames.getFirstName("Bergoglio"));
-        assertEquals(firstNames.getFirstName("Francesco Cadili"), "Francesco");
-        // This Philip Seymour Hoffman example is from ShibUtilTest.
-        assertEquals("Philip Seymour", firstNames.getFirstName("Philip Seymour Hoffman"));
-
-        // test Smith (is also a name)
-        assertEquals("John", firstNames.getFirstName("John Smith"));
-        // resolved using hint file
-        assertEquals("Guido", firstNames.getFirstName("Guido van Rossum"));
-        // test only name
-        assertEquals(firstNames.getFirstName("Francesco"), "Francesco");
-        // test only family name
-        assertEquals(firstNames.getFirstName("Cadili"), null);
-    }
-
-    /**
-     * Name is composed of: The string is composed of:
-     * <Family Name>, <First Names>
-     */
-    @Test
-    public void testNameConvention2() {
-        assertEquals(firstNames.getFirstName("Awesome, Audrey"), "Audrey");
-        assertEquals(firstNames.getFirstName("Bergoglio, Jorge Mario"), "Jorge Mario");
-        assertEquals(firstNames.getFirstName("Cadili, Francesco"), "Francesco");
-        assertEquals("Guido", firstNames.getFirstName("van Rossum, Guido"));
+    @ParameterizedTest
+    @CsvSource(delimiter = '|', value = {
+            "Awesome, Audrey|Audrey|Awesome",
+            "Bergoglio, Jorge Mario|Jorge Mario|Bergoglio",
+            "Cadili, Francesco|Francesco|Cadili",
+            "Bergoglio|''|Bergoglio",
+            "Francesco Cadili|Francesco|Cadili",
+            "Philip Seymour Hoffman|Philip Seymour|Hoffman",
+            "John Smith|John|Smith",
+            "Guido van Rossum|Guido|van Rossum",
+            "Francesco|Francesco|''",
+            "Cadili|''|Cadili",
+            "Jorge Mario Bergoglio|Jorge Mario|Bergoglio",
+            "Jorge Bergoglio Mario Luigi|Jorge|Bergoglio Mario Luigi" // this works differently
```

Comment:
`// this works differently` was just to point that this behaves different than previous solution. Without that context it doesn't have any value and could be removed